packages/block-editor/src/components/url-input/index.js

```diff
@@ -258,7 +250,6 @@ class URLInput extends Component {
 		if (
 			value &&
 			! disableSuggestions &&
-			! this.state.isUpdatingSuggestions &&
```

Comment:
This `onFocus` handler is the only place where `state.isUpdatingSuggestions` does something useful: it prevents the REST request from being triggered on focus when there is another one already running.

Could be replaced with checking `this.suggestionsRequest === null`. But first we need to start resetting `this.suggestionsRequest` back to `null` when the request is finished. That's not currently done.

Reply:
I've used `Promise.finally()` to reset `this.suggestionsRequest` to `null` and then conditionalised the focus logic based on that.